src/docfx.website.themes/common/RestApi.common.js

```diff
@@ -22,6 +22,29 @@ exports.transform = function (model) {
             resolveAllOf(child);
             transformReference(child);
         };
+        if (!model.tags || model.tags.length == 0) {
```

Comment:
according to [http://www.w3schools.com/js/js_best_practices.asp](http://www.w3schools.com/js/js_best_practices.asp), should avoid ==

---
In reply to: [68890788](https://github.com/dotnet/docfx/pull/442#discussion_r68890788) [](ancestors = 68890788,68890026)